docs/EN/Configuration/transition-qa.md

```diff
@@ -1,127 +1,161 @@
-once you get the settings right.
 
-### What does AndroidAPS have that Open-iAPS doesn't?
+## Coming From AndroidAPS
+AndroidAPS and Open-iAPS are based on the same algorithm (Oref) and base system (Artificial Pancreas System, APS). While there are several differences between AndroidAPS and Open-iAPS, the main algorithm is the same, 
+meaning you can expect Open-iAPS to behave the same way AndroidAPS does generally.
 
-* Custom automations
+<details>
+  <summary><b>Benefits of AndroidAPS</b></summary>
+
+* Custom automation
 * Detailed reporting
 * Working remote bolus (for caregivers, not working at the moment in Open-iAPS)
 * A built-in learning program
-
-### What does Open-iAPS have that AndroidAPS doesn't?
+</details>
+<br>
+<details>
+<summary><b>Benefits of Open-iAPS</b></summary>
 
 * Open-iAPS works on your iPhone and Apple Watch
-* Open-iAPS has dynamic ISF and dynamic IC support in main branch
-
-### Why should I switch from AndroidAPS to Open-iAPS?
-If you have been successfull with AndroidAPS but miss the Apple ecosystem, you can easily transition to Open-iAPS. If you've been struggling to get stable BG with AndroidAPS, you will probably have the same issues with Open-iAPS.
-Take some time to read the docs and fine-tune your settings. Join Discord or Facebook to get some help from other users. Once you've got your settings in order, you can consider making a switch to Open-iAPS.
-
---
-
-## Coming from iOS Loop
-iOS Loop uses different algorithms, meaning it has a very different approach than Open-iAPS. If you're coming from iOS Loop, you will have to change the way you think. Please read that statement again three times, as it is really important.
-
-### What does Loop have that Open-iAPS doesn't?
+* Open-iAPS has dynamic CR support
+</details>
+<br>
+<details>
+  <summary><b>Should I switch from AndroidAPS to Open-iAPS?</b></summary>
+- If you have been successful with AndroidAPS but prefer the Apple ecosystem, you can easily transition to Open-iAPS. - If you've struggled to get stable BG with AndroidAPS, you may have the same issues with Open-iAPS.
+  
+Take some time to read the docs and fine-tune your settings. To get help from other users, join [Discord](https://discord.gg/dbe5Twav8D) or [Facebook](https://www.facebook.com/groups/1351938092206709/). Once your settings are in order, consider switching to Open-iAPS.
+</details>
+<br>
+
+## Coming From iOS Loop
+iOS Loop uses different algorithms, meaning it has a different approach than Open-iAPS. If you're coming from iOS Loop, you must change how you think. Please read that statement again three times, as it is crucial.
+
+<details>
+  <summary><b>Benefits of Loop</b></summary>
 
 * A very clean, minimalistic user interface
 * A built-in onboarding guide
-* Few settings
-* A dedicated Follow app for caregivers
-* A large community of people ready to help
-
-### What does Open-iAPS have that Loop doesn't?
+* Simple settings
+* A dedicated Loop Follow app for caregivers
```

Comment:
The LoopFollow app supports both Loop and iAPS and Open-iAPS.
The dedicated app for caregivers is called LoopCaregiver.

Reply:
I suggest you fold in here that any system that uploads data to Nightscout (Loop, Open-iAPS, AAPS, etc.) can use LoopFollow to monitor and provide alarms on an iPhone. The documentation for LoopFollow is at https://www.loopandlearn.org/loop-follow/